CMakeLists.txt

```diff
@@ -59,6 +59,10 @@ project(CUTLASS VERSION ${_CUTLASS_VERSION_MAJOR}.${_CUTLASS_VERSION_MINOR}.${_C
 
 ################################################################################
 
+if (CUTLASS_ENABLE_SYCL)
+  set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --allow-unsupported-compiler")
```

Comment:
What's the motivation for this? How are CMAKE_CUDA_FLAGS used with CUTLASS_ENABLE_SYCL?